File: backend/internal/domain/reality_tv.go
package domain

type RealityTVDashboardMetrics struct {
	TotalSeasons        int `json:"totalSeasons"`
	ActiveSeason        any `json:"activeSeason"`
	TotalApplications   int `json:"totalApplications"`
	PendingApplications int `json:"pendingApplications"`
	TotalContestants    int `json:"totalContestants"`
	ActiveVotingRounds  int `json:"activeVotingRounds"`
	TotalVotes          int `json:"totalVotes"`
	PaidVotes           int `json:"paidVotes"`
	FreeVotes           int `json:"freeVotes"`
	OpenTickets         int `json:"openTickets"`
}

File: frontend-web/src/types/realityTv.ts
export type RealityTVActiveSeason = {
  season_title: string;
  season_number: number;
  status: string;
} | null;

export type RealityTVDashboardMetrics = {
  totalSeasons: number;
  activeSeason: RealityTVActiveSeason;
  totalApplications: number;
  pendingApplications: number;
  totalContestants: number;
  activeVotingRounds: number;
  totalVotes: number;
  paidVotes: number;
  freeVotes: number;
  openTickets: number;
};
